frontend/components/ticket/Ticket.vue

```diff
@@ -35,7 +35,7 @@
           <div class="flex flex-wrap mb-2" v-else>
             No labels selected
           </div>
-          <label-dropdown :selected="labels" :values="inbox.labels" v-if="canShare" v-model="labels"
+          <label-dropdown :selected="labels" :values="inbox.labels" v-if="inbox && canShare" v-model="labels"
```

Comment:
This fixes the error: `Error in render: "TypeError: Cannot read property 'labels' of null"`